src/qml/Statusbar.qml

```diff
@@ -289,9 +291,18 @@ Item {
             iconSize: Theme.itemHeightExtraSmall
             source: "image://theme/icon_gps_normal"
         }
+
         StatusbarItem {
             iconSize: Theme.itemHeightExtraSmall
-            source: "image://theme/icon_play_pause"
+            source: (mprisManager.currentService && mprisManager.playbackStatus == Mpris.Playing) ?
+                        "/usr/share/themes/glacier/fontawesome/icons/pause.png"
+                      : "/usr/share/themes/glacier/fontawesome/icons/play.png"
```

Comment:
Here same with image://theme...
I didn't yet have time to test this PR, did you @locusf manage?

Reply:
Doh....i fix it evening sorry

Reply:
NP:)
How to try this PR? Install glacier-music and have some songs?
I will try today evening or tomorrow.

Reply:
Yeap :)

Reply:
Nope, didn't have time to test this yet.